scripts/popup.js

```diff
@@ -182,21 +177,40 @@ $(window).ready(() => {
     $(`#section-${sectionName}`).toggle();
   }
 
+  function applyCustomScrapedData(scrapeData) {
+    console.log(scrapeData);
+    const scrapeConfig = Archivist.getScrapperConfig(scrapeData.url);
+
+    Object.keys(scrapeData.fields).forEach((popupFieldId) => {
+      const curFieldOptions = scrapeConfig[popupFieldId];
+      const curFieldValue = scrapeData.fields[popupFieldId];
+      if (curFieldOptions.isCorrectFormat) {
+        $(`#${popupFieldId}`).val(curFieldValue);
+      } else {
+        console.log('formatted');
+        const formattedVal = curFieldOptions.dataFormatFunc(curFieldValue);
+        console.log(formattedVal);
+        $(`#${popupFieldId}`).val(formattedVal);
+      }
+    });
+  }
+
   // Scrapes page and sets some metadata fields
   function scrapePage() {
```

Comment:
This function doesn't scrape the page really, it sets the default values for the metadata fields.